code/modules/exploration_crew/exploration_laser_gun.dm

```diff
@@ -29,11 +29,10 @@
 	muzzle_type = /obj/effect/projectile/muzzle/laser
 	impact_type = /obj/effect/projectile/impact/laser
 
-/obj/item/projectile/beam/laser/anti_creature/on_hit(atom/target, blocked)
-	damage = initial(damage)
-	if(!iscarbon(target) && !issilicon(target))
-		damage = 30
-	. = ..()
+/obj/item/projectile/beam/laser/anti_creature/prehit_pierce(atom/target)
+    if(!iscarbon(target) && !issilicon(target))
+        damage += 15
```

Comment:
this will permanently increase damage

Reply:
it's on the projectile and not the actual gun but i think it should directly assign to 30 instead of doing addition or even just multiply by 2

Reply:
changed to damage = 30.

I dont think it did permentantly increase damage as each time a projectile was made the damage value was set to 15, from what I could tell, and only on hitting the value was increased but if this is better I'll set it to this.